src/main/query_profiler.cpp

```diff
@@ -141,6 +134,22 @@ void QueryProfiler::EndQuery() {
 	}
 	this->is_explain_analyze = false;
 }
+string QueryProfiler::PrintAsString() const {
```

Comment:
Perhaps we could turn this method into `ToString()`, and rename the `ToString()` method to `ToQueryTree()`? That might be more clear.

Reply:
Sure. I wasn't sure of the semantics of ToString and whether you always want a human readable version. WIll change.

Reply:
@Mytherin The ToString makes it optional to print out the optimizer output and also it prints more things, like total time. Next to that, it is a sibling to the `ToStream` method. Do you still feel you want this change? (i.e., rename this method to be the `ToString`, and rename current `ToString`, and probably its sibling, to something else)

Reply:
I think it makes sense for the `ToString` to take the print profiler method into account, so the rename makes sense to me. `ToStream` should then indeed also be renamed to `QueryTreeToStream` or so.